src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const ReparedRobot = {};
+
+  for (const key in robot) {
+    if (ReparedRobot[robot[key]]) {
```

Comment:
better 2 check 4 null before creating new robot

Reply:
agree

Reply:
Better use hasOwnProperty to check is property in object. because in will also return true if key gets found somewhere in the prototype chain, otherwise Object.hasOwnProperty (like the name already tells us), will only return true if key is available on that object directly (its "owns" the property).